### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by saniales** [fastio/pedis#25]
I see no activity from November 2017
Is this project dead? is it still compatible with latest Redis version?

**Comment by fastio** [comment]
hi @saniales, I have changed my job, and there is no more time for developing this project now. But is not dead, I will continue to update it in the future. Thanks.

**Comment by saniales** [comment]
Thank you, and what is the latest Redis compatible version? I want to use pedis in my environment but my bosses are asking. This is for sure an awesome swiis knife tool for efficiency :)

**Comment by fastio** [comment]
I agree what you said.  It's not recommended to use it in production now.  It should be fully tested before using it in production.

**Comment by tracymacding** [comment]
i want to make some contribution if possible~

**Comment by fastio** [comment]
Welcome
